rules-tests/Transform/Rector/Class_/AddAllowDynamicPropertiesAttributeRector/config/configured_rule.php

```diff
@@ -7,5 +7,10 @@
 
 return static function (ContainerConfigurator $containerConfigurator): void {
     $services = $containerConfigurator->services();
-    $services->set(AddAllowDynamicPropertiesAttributeRector::class);
+    $services->set(AddAllowDynamicPropertiesAttributeRector::class)
```

Comment:
please add 1 more test class with no config to fallback to change all classes, even without namespace